### DISCUSSION TITLE
Loggers don't work with Pytest

### DISCUSSION THREAD

**Discussion by harmin-parra** [pytest-dev/pytest#8075] [General] [answered]
The Python logging feature doesn't work with Pytest

This code doesn't work in Pytest:

`import logging`
`logging.basicConfig(filename='example.log', level=logging.DEBUG)`
`logging.debug("Hello World")`

**Comment by RonnyPfannschmidt** [comment, marked as answer]
https://docs.pytest.org/en/stable/logging.html 

the log level for pytest testing has to be explicitly set for the testrunner , for example with the --log-level option
